apps/ledger-live-mobile/src/families/cosmos/RedelegationFlow/01-SelectValidator.tsx

```diff
@@ -81,8 +82,7 @@ function RedelegationSelectValidator({ navigation, route }: Props) {
       ),
     [delegations, transaction.sourceValidator],
   );
-  invariant(srcDelegation, "source delegation required");
-  const max = srcDelegation.amount;
+  const max = srcDelegation?.amount;
```

Comment:
Unimportant, but maybe a little bit more readable if you set the default `BigNumber(0)` value here instead of below.

Reply:
I tried to do || new BigNumber(0) but because of hook I can't really do it here